javascript-source/discord/handlers/streamHandler.js

```diff
@@ -345,6 +360,15 @@
                 $.inidb.set('discordSettings', 'onlineChannel', channelName);
                 $.discord.say(channel, $.discord.userPrefix(mention) + $.lang.get('discord.streamhandler.channel.set', channelName));
             }
+
+            /**
+             * @discordcommandpath streamhandler toggledeletemessage - Toggles if online announcements get deleted after stream.
+             */
+            if (action.equalsIgnoreCase('toggledeletemessage')) {
+                deleteMessageToggle = !deleteMessageToggle;
+                $.inidb.set('discordSettings', 'deleteMessageToggle', deleteMessageToggle);
+                $.discord.say(channel, $.discord.userPrefix(mention) + $.lang.get('discord.streamhandler.delete.toggle', (onlineToggle === true ? $.lang.get('common.enabled') : $.lang.get('common.disabled'))));
```

Comment:
`onlineToggle` should be changed to `deleteMessageToggle `

Reply:
Good catch, thanks.